docs/src/_includes/layouts/doc.html

```diff
@@ -148,4 +148,27 @@ <h1>{{ title }}</h1>
     </a>
 </div>
 
+<div id="copy-announcement" class="visually-hidden" aria-live="polite" aria-atomic="true"></div>
+
+<script>
+    (function() {
+        const copyBtns = document.querySelectorAll("#copyBtn");
+        const announcementRegion = document.getElementById("copy-announcement");
+
+        for (const copyBtn of copyBtns) {
+            const copyText = copyBtn.previousElementSibling.firstChild.cloneNode(true).textContent;
```

Comment:
```suggestion
            const copyText = copyBtn.previousElementSibling.firstChild.textContent;
```
Why do we need to clone and then copy the textContent? Can't we directly access the textContent instead?